web_src/css/base.css

```diff
@@ -518,6 +520,7 @@ a.label,
 .ui.menu .item {
   color: var(--color-text);
   user-select: auto;
+  line-height: var(--line-height-default); /* fomantic uses "1" which causes overflow problems because "1" doesn't consider the descent part */
 }
```

Comment:
Also here, maybe we should adjust vertical padding.

Reply:
Fixed in https://github.com/go-gitea/gitea/pull/26553/commits/9ba7096ba456dbf9910ee6477a3ca018308d57a8.